gradle.properties

```diff
@@ -1,4 +1,4 @@
 org.gradle.caching=true
 group=io.ballerinax
-version=7.2.0
-ballerinaLangVersion=2201.4.0-20230124-005500-f19b94e2
+version=7.4.1
```

Comment:
Any specific reason to go for `7.4.1` instead of `7.2.1`?

Reply:
already 7.4.0 is released and it's not updated in the master branch